overrides/main.html

```diff
@@ -154,5 +154,6 @@
 {{ super() }} 
 
 <script src="javascripts/feedback.js"></script>
+<script src="javascripts/runllm-widget.js"></script>
```

Comment:
Are the javascripts included from the mkdocs even running, or does this just override everything?

If this is how you wish to include the runllm widget script, we can remove the runllm-widget.js script and entry in mkdocs.yml and replace the line above with the following:

```html
<script
  defer
  id="runllm-widget-script"
  type="module"
  src="https://cdn.jsdelivr.net/npm/@runllm/search-widget@stable/dist/run-llm-search-widget.es.js"
  version="stable"
  runllm-preset="mkdocs"
  runllm-server-address="https://api.runllm.com"
  runllm-assistant-id="111"
  runllm-position="BOTTOM_RIGHT"
  runllm-keyboard-shortcut="Mod+j"
  runllm-slack-community-url="https://quix.io/slack-invite"
  runllm-name="RunLLM"
></script>
```

Reply:
@agiron123 Thanks -  It was my assumption (and perhaps I am wrong) that you had to update the override AND `mkdocs.yml` if you wanted the new Javascript to kick in or load it at a specific point. I will try your solution anyway.

Reply:
It should just be one I believe, but it's hard to say definitively since people can set up their docs sites differently.